<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProtoWindow</class>
 <widget class="QMainWindow" name="ProtoWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ProtoWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout" rowstretch="10,1,1" columnstretch="0,0,0,0,0">
    <item row="1" column="0" colspan="5">
     <widget class="QPlainTextEdit" name="toPutPlainTextEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="backPushButton">
      <property name="text">
       <string>&lt;--</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="showPushButton">
      <property name="text">
       <string>Show</string>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QPushButton" name="forwardPushButton">
      <property name="text">
       <string>--&gt;</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="5">
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1004</width>
         <height>601</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="rotateLeftPushButton">
      <property name="text">
       <string>90° nach links drehen</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="rotateRightPushButton">
      <property name="text">
       <string>90° nach rechts drehen</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
